site/_scss/blocks/_code-sections.scss

```diff
@@ -99,6 +99,10 @@
   color: var(--color-pink-medium);
 }
 
+.code-sections__deprecated {
+  color: var(--color-red-medium);
```

Comment:
Out of scope for this PR, but should we have safe fallbacks in the event that CSS custom properties aren't supported?

Reply:
We use a _lot_ of CSS Custom Properties. This was Rob's design decision.